src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -131,7 +131,7 @@
     <value>The data store operation failed.</value>
   </data>
   <data name="DicomElementValidationFailed" xml:space="preserve">
-    <value>Dicom element '{0}' with value '{1}' does not validate VR '{2}': {3}.</value>
+    <value>Dicom element '{0}' does not validate VR '{1}': {2}.</value>
```

Comment:
should we say something along the lines of given VR does not match expected VR type?

Reply:
As my understanding, "does not match expected VR type" means customer provided wrong PR. And in our case, the validation could failure a bunch of different reason, e.g: provided value is too long.

Reply:
Perhaps then just `Dicom element '{0}' failed validation for VR '{1}': {2}` ?